src/sentry/incidents/subscription_processor.py

```diff
@@ -630,8 +630,8 @@ def trigger_alert_threshold(
             if not self.active_incident:
                 detected_at = self.calculate_event_date_from_update_date(self.last_update)
                 activation: AlertRuleActivations | None = None
-                if self.alert_rule.monitor_type == AlertRuleMonitorType.ACTIVATED:
-                    activations = list(self.subscription.alertruleactivations_set)
+                if self.alert_rule.monitor_type == AlertRuleMonitorType.ACTIVATED.value:
+                    activations = list(self.subscription.alertruleactivations_set.all())
```

Comment:
this is another functional change that was missed

cannot `list`ify the `_set` - but still need to grab `.all()` in order to `list`ify